_platforms.rst

```diff
@@ -0,0 +1,141 @@
+.. _platforms:
+
+Publishing Platforms
+=====================
+
+Three platforms facilitate publication and discovery of all DM documentation: LSST the Docs, DocuShare, and LSST DocHub.
+Publishing all documentation in this system ensures consistency in presentation and discovery.
+This section summarizes the functionality of these platforms for policies for their use.
+
+.. _platforms-ltd:
+
+LSST the Docs
+-------------
+
+LSST the Docs (LTD) is a platform for continuously publishing versioned documentation to the web.
+LTD is more fully described in `SQR-006`_; please see that technote for details beyond the scope of this summary.
+
+All DM documentation projects maintained in a version control system (specifically, Git with GitHub_) are published with LTD.
+Other documents are published with :ref:`DocuShare, see below <docushare>`.
+
+LSST the Docs's key features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following LTD features are key to the DM Documentation Architecture:
+
+- **LTD is a service that hosts static websites,** that is, websites that do not require server-side rendering logic.
+  Supporting only static websites ensures that LTD is reliable and scalable.
+  This design decision also coerces documentation into a format that can be readily archived, which is important to DM's scientific legacy.
+
+- **Each documentation project hosted by LTD is published from a unique subdomain of the lsst.io domain.**
+  This helps to make human-friendly URLs.
+  Audiences should see the ``lsst.io`` domain as a brand synonymous with LSST documentation.
+
+- `LTD publishes multiple versions of a document <https://sqr-006.lsst.io/#versioned-documentation-urls>`_, each under a different URL path prefix.
```

Comment:
I do mean prefix (for the path, not domain), as in

`/v/v1/`...

or `/v/DM-9999/` ...